터미널 환경설정 레이아웃 디자인

RBJH opened the pull request:
터미널 환경설정 레이아웃 설계
- UX를 고려하여 Stepper를 활용한 환경설정 과정 정의
- 의미에 맞는 태그 설정을 위한 토론
- 이슈에 작업사항 리스트 작성

RBJH committed:
Feat: 터미널 환경설정 창 구조 설계

전체적인 레이아웃을 구성하기 위해 헤더 및 내용물을 구분하여 설계

RBJH committed:
Feat: 터미널 환경설정 헤더와 내용물 부분 구조 설계

헤더에는 터미널 환경설정 창을 알리기 위한 제목과
내용물 부분에는 각 환경설정의 단계를 알리기 위한 스테퍼 및
각 스텝의 선택 및 스템을  이동하기 위한 버튼을 구성하는 footer로 구성

RBJH committed:
Feat: 터미널 환경설정 stepper 디자인

각 스텝을 표현할  아이콘을 위해  @fortawesome 패키지 설치
스텝 간격에 대한 레이아웃을 위해 justify-content 속성 사용
각 스텝을 렌더링 하기 위해 리스트와 키를 활용한 React의 map 사용

dimsssss reviewed (commented):
고생하셨습니다

ImuruKevol commented on client/src/components/editor/terminal-setting-drawers/Stepper.jsx:
```diff
@@ -0,0 +1,22 @@
+const Stepper = () => {
+  const getSteps = () => {
+    return Object.values(STEP_TYPE).map((icon, index) => {
+      return <Step icon={icon} key={index} />;
```
key에는 uuid 모듈을 사용하시면 좋습니다!

RBJH committed:
Feat: 터미널 환경설정 내용물 레이아웃 설정

내용물을 header, main, footer 의미에 맞게 태그 설정
각 내용물에 대해 이해하기 쉽도록 title과 list로 노출
버튼은 UX를 고려한 투표를 통해 중앙에 위치
